dev.php

```diff
@@ -1,12 +1,16 @@
 <?php
 define('DAHL_DEVROOT', dirname(__file__));
-if (
-    strpos($_SERVER['SCRIPT_NAME'], 'composer') !== false
-    || strpos($_SERVER['SCRIPT_NAME'], 'magerun') !== false
-    || strpos($_SERVER['SCRIPT_NAME'], 'cron.php') !== false
-) {
```

Comment:
Thanks, this is certainly helping. 
We still need to keep the blacklist for scripts like `cron.php`, `install.php` though.